wp-content/themes/amplify/functions.php

```diff
@@ -45,3 +45,91 @@ function largo_parent_theme_enqueue_styles() {
 	);
 }
 add_action( 'wp_enqueue_scripts', 'largo_parent_theme_enqueue_styles', 20 );
+
+/**
+ * Add query vars specific to the Amplify child theme
+ * 
+ * @param array $vars The array of available query vars
+ * 
+ * @return array $vars The modified array of available query vars
+ */
+function amplify_add_query_vars( $vars ) {
+	
+	$vars[] = 'amplify-feed';
+	
+	return $vars;
+
+}
+add_filter( 'query_vars', 'amplify_add_query_vars' );
+
+/**
+ * Load the Amplify embed feed template if specific query param is present
+ * 
+ * @param str $template The path of the current template
+ * 
+ * @return str $template The path of the template to include
+ */
+function amplify_load_feed_template( $template ) { 
+
+    if ( ! is_admin() ) {
+        // if the params are set, use our amplify feed template if it exists
+		if( isset( $_GET['amplify-feed'] ) && locate_template( 'amplify-feed.php' ) ){
+			return get_stylesheet_directory().'/amplify-feed.php';
+		// else, continue with whatever template was being loaded
+		} else {
+			return $template;
+		}
+    }
+	return $template;
+	
+}
+add_filter( 'template_include', 'amplify_load_feed_template' );
+
+/**
+ * Add the "More link" input field to the tag edit page for 'lr-tags' taxonomy
+ * 
+ * @param obj $context The wp term that is being modified
+ */
+function amplify_add_saved_link_roundups_more_link_metabox( $context = '' ) {
+
+	if( 'lr-tags' == $context->taxonomy ){
+		
+		$term_more_link = get_term_link( $context, $context->taxonomy );
```

Comment:
This isn't going to be a useful default because the `lr-tags` taxonomy doesn't have public archives.  But there also isn't any other good default. Maybe this should be blank to begin with?

Reply:
https://github.com/INN/umbrella-inndev/pull/130/commits/37305df5350a65458d8eaa16730acc774dca60f8 updates it so there's no default and https://github.com/INN/umbrella-inndev/pull/130/commits/1bce875199d80019f00dcebe6e7ba85304e3c38b updates the frontend to only display the `Learn More` link if it exists instead of defaulting to the tag term link.